runtime/doc/faq.txt

```diff
@@ -0,0 +1,822 @@
+`termguicolors` is set.
+
+'guicursor' gives an example, but here's a more complicated example
+which sets different colors in insert-mode and normal-mode:
+
+>vim
+    :set termguicolors
+    :hi Cursor guifg=green guibg=green
+    :hi Cursor2 guifg=red guibg=red
+    :set guicursor=n-v-c:block-Cursor/lCursor,i-ci-ve:ver25-Cursor2/lCursor2,r-cr:hor20,o:hor50
+<
+
+
+CURSOR STYLE ISN'T RESTORED AFTER EXITING OR SUSPENDING AND RESUMING NVIM ~
+
+Terminals do not provide a way to query the cursor style. Use autocommands to
+manage the cursor style:
+
+>vim
+    au VimEnter,VimResume * set guicursor=n-v-c:block,i-ci-ve:ver25,r-cr:hor20,o:hor50
+      \,a:blinkwait700-blinkoff400-blinkon250-Cursor/lCursor
+      \,sm:block-blinkwait175-blinkoff150-blinkon175
+
+    au VimLeave,VimSuspend * set guicursor=a:block-blinkon0
+<
+
+
+CURSOR SHAPE DOESN'T CHANGE IN TMUX ~
+
+tmux decides that, not Nvim. See |tui-cursor-shape| for a fix.
+
+See #3165 https://github.com/neovim/neovim/pull/3165 for discussion.
+
+
+CURSOR FLICKER IN TMUX? ~
+
+If cursor `_` appears and disappears very quickly when opening nvim without a
+document under tmux, and you set `ctermbg` in `EndOfBuffer` and `Normal`, try
+setting these to `NONE`:
+
+>vim
+    hi EndOfBuffer ctermbg=NONE ctermfg=200 cterm=NONE
+    hi Normal ctermbg=NONE ctermfg=200 cterm=NONE
+<
+
+
+HOW TO USE THE WINDOWS CLIPBOARD FROM WSL? ~
+
+**Update:** Try the suggestion given by |clipboard-wsl|:
+
+>vim
+    let g:clipboard = {
+      \   'name': 'WslClipboard',
+      \   'copy': {
+      \      '+': 'clip.exe',
+      \      '*': 'clip.exe',
+      \    },
+      \   'paste': {
+      \      '+': 'powershell.exe -c [Console]::Out.Write($(Get-Clipboard -Raw).tostring().replace("`r", ""))',
+      \      '*': 'powershell.exe -c [Console]::Out.Write($(Get-Clipboard -Raw).tostring().replace("`r", ""))',
+      \   },
+      \   'cache_enabled': 0,
+      \ }
+<
+
+
+OLD INSTRUCTIONS...
+
+To use the Windows clipboard from within WSL, `win32yank.exe`
+https://github.com/equalsraf/win32yank has to be on our `$PATH`.
+
+If Nvim is installed on both Windows and within the WSL distribution, the
+`win32yank.exe` binary provided by the Nvim Windows installation can be
+symlinked to a directory included in our `$PATH` so it can be found by Nvim
+on WSL. Replace `$NEOVIM_WIN_DIR` with the path to our Nvim Windows
+installation, e.g. `/mnt/c/Program Files/Neovim`. The command can then be
+symlinked using:
+
+>vim
+    sudo ln -s "$NEOVIM_WIN_DIR/bin/win32yank.exe" "/usr/local/bin/win32yank.exe"
+<
+
+On some versions of Windows, WSL is unable to execute symbolic links to Windows
+executables (microsoft/WSL#3999
+https://github.com/microsoft/WSL/issues/3999). In that case, use one of the
+methods in #12113 (comment)
+https://github.com/neovim/neovim/issues/12113#issuecomment-612827446.
+
+If Nvim is only installed within our WSL distribution, we can just install
+`win32yank.exe` manually:
+
+>vim
```

Comment:
shouldn't this be `>bash`

Reply:
Yes, I plan to go through the document and fix things (and remove outdated information) sometime today.